tailwind.config.js

```diff
@@ -21,5 +21,5 @@ module.exports = {
       },
     },
   },
-  plugins: [require("tailwind-scrollbar")],
+  plugins: [require("tailwind-scrollbar"), require("@tailwindcss/typography")],
```

Comment:
What does this do?

Reply:
Does this also added the changes to the other components of the html. Since I am assuming some default styles has to be applicable to the entire UI for it to look good. From designing the wireframes coding the actual designs, I kept tailwind as a base so make sure it doesn't hamper the design

Reply:
No it doesn't ... It provides a className named "prose" for us to use on parent tag in which we want to respect the default HTML styling ..

In this case we're using this class on editor and the post title render div